helm/templates/daemonset.yaml

```diff
@@ -0,0 +1,40 @@
+{{- if .Values.lldpd.enabled }}
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: "{{ .Release.Name }}-lldpd"
+spec:
+  selector:
+    matchLabels:
+      name: lldpd
+  template:
+    metadata:
+      labels:
+        name: lldpd
+    spec:
+      tolerations:
+      - key: node-role.kubernetes.io/control-plane
+        operator: Exists
+        effect: NoSchedule
+      - key: node-role.kubernetes.io/master
+        operator: Exists
+        effect: NoSchedule
+      containers:
+      - name: lldpd
+        image: "{{ .Values.lldpd.image.repository }}:{{ .Values.lldpd.image.tag }}"
+        imagePullPolicy: {{ .Values.lldpd.image.pullPolicy }}
+        securityContext:
+          capabilities:
```

Comment:
this looks fine to me. I tried it on another setup where they are coming up fine. Just a quick thing if we want to harden the security more by dropping all others and allowing only these two:

```
          capabilities:
            drop: ["ALL"]
            add: ["NET_ADMIN", "NET_RAW"]